android/FacebookSDK400/build.gradle

```diff
@@ -1,23 +1,27 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
-buildscript {
-    project.apply from: "${rootDir}/constants.gradle"
 
+buildscript {
+    
     repositories {
-        jcenter()
-        mavenCentral()
         google()
+        jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:${project.ext.gradleVersion}"
+        classpath 'com.android.tools.build:gradle:3.1.3'
```

Comment:
does the gradle version have to be this ?

Reply:
Yes, it is ok, the variable `${project.ext.gradleVersion}` traced by git is from different project.

Reply:
yes Alex, understood. merge already